package.json

```diff
@@ -13,13 +13,14 @@
     "babel-plugin-transform-es2015-destructuring": "^6.6.5",
     "babel-preset-es2015": "^6.6.0",
     "browser-sync": "^2.12.10",
-    "browserify": "^13.0.1"
+    "browserify": "^13.0.1",
+    "remarkable": "^1.6.2"
   },
   "scripts": {
     "start": "scripts/server.js",
-    "deploy": "npm run babel && npm run browserify && git subtree push --prefix site origin gh-pages",
+    "deploy": "npm run babel && npm run browserify && node site/scripts/generate_readme.js && cp ./site/*.html ./site-gh-pages && cp ./site/*.css ./site-gh-pages && cd site-gh-pages && git add . && git commit -m \"Update gh-pages\" && git push origin gh-pages && cd ..",
```

Comment:
To answer your question from earlier, I think it should be a standalone npm task. It does what it says it will do, and it is very useful for officially updating the page. But I have my doubts on that answer. For one, committing and pushing changes with three words makes me a little uneasy, but actually I wouldn't be running `npm run deploy` if I didn't fully intend on committing the changes to `gh-pages` Really the changes should be committed to master first. This just compiles it into its necessary parts and shoots it up to `gh-pages`.